priv/riak_kv.schema

```diff
@@ -256,6 +256,35 @@
   hidden
 ]}.
 
+%% @doc Tictacaae - change log level of parallel keystore
+%% Backend logging within leveled is relatively verbose, so in some cases
+%% there may be a preference of running leveled at warning not info.  This will
+%% only change logging in parallel-mode aae backends, not of leveled when
+%% running as a vnode backend
+{mapping, "tictacaae_backendloglevel", "riak_kv.tictacaae_backendloglevel", [
+  {datatype, {enum, [debug, info, warning]}},
+  {default, info}
+]}.
+
+%% @doc Tictacaae - change log level of tictacaae
+%% Set the log level for each aae_controller, and also for aae exchanges.  It
+%% is strongly recommended to keep this at info.
+{mapping, "tictacaae_loglevel", "riak_kv.tictacaae_loglevel", [
+  {datatype, {enum, [info, warning]}},
+  {default, info},
+  hidden
+]}.
+
+%% @doc Tictacaae - change compression method of parallel keystore
+%% Better performance is expected when using the NIF-based zstd compression. 
+%% Actual compression achieved is similar to native (zlib).  Native will
+%% compress using the built-in compression option for Erlang term_to_binary.
+%% This will only change compression in parallel-mode aae backends, not of
+%% leveled when running as a vnode backend
+{mapping, "tictacaae_backendcompression", "riak_kv.tictacaae_backendcompression", [
+  {datatype, {enum, [native, zstd]}},
```

Comment:
In testing, we haven't found situations where lz4 is better than zstd.  I'm therefore considering deprecating it in the future, so I though it would be wise if we're adding new compression option to only offer native (for those nif-averse) and zstd (for those performance sensitive)